build/Build.Steps.Windows.cs

```diff
@@ -256,13 +256,35 @@ void BuildDockerImage(Project project)
 
     private void RunNetFxGacOperation(string operation)
     {
+        // To update the GAC, we need to run the tool as Administrator.
+        // Throw if not running as a Windows Administrator.
+        if (!IsWindowsAdministrator())
+        {
+            throw new InvalidOperationException("This target must be run on Windows as Administrator.");
+        }
+
         var netFxAssembliesFolder = TracerHomeDirectory / MapToFolderOutput(TargetFramework.NET462);
         var installTool = Solution.GetProjectByName(Projects.Tools.GacInstallTool);
 
         DotNetRun(s => s
             .SetProjectFile(installTool)
             .SetConfiguration(BuildConfiguration)
             .SetApplicationArguments($"{operation} {netFxAssembliesFolder}"));
+
+        static bool IsWindowsAdministrator()
+        {
+            if (!IsWin)
```

Comment:
This task is only called on Windows but for completeness I'm also including this test here.